Common/Securities/Volatility/StandardDeviationOfReturnsVolatilityModel.cs

```diff
@@ -131,16 +152,25 @@ public override IEnumerable<HistoryRequest> GetHistoryRequirements(Security secu
                 configuration.DataTimeZone);
             var utcStartTime = localStartTime.ConvertToUtc(security.Exchange.TimeZone);
 
+            var bar = security.GetLastData();
+            if (bar == null)
+            {
+                bar = typeof(TradeBar).GetBaseDataInstance();
+                bar.Symbol = security.Symbol;
+            }
```

Comment:
> var bar = security.GetLastData();

This seems a bit indirect, what if there was custom data in it? Suggest getting the data type from the actual subscriptions configurations above, could check for TradeBar/QuoteBar being in it, same comment for `QCAlgorithm.Universe.cs`.

I think `RelativeStandardDeviationVolatilityModel` & `StandardDeviationOfReturnsVolatilityModel` should use the same implementation of `GetHistoryRequirements`. Think that `RelativeStandardDeviationVolatilityModel` should also allow specifying some custom resolution for the history request, should be standarized.